types/shelljs/index.d.ts

```diff
@@ -97,7 +97,7 @@ export function mkdir(options: string, ...dir: Array<string | string[]>): void;
  * @param   path   The path.
  * @return        See option parameter.
  */
-export function test(option: string, path: string): boolean;
+export function test(option: "-b" | "-c" | "-d" | "-e" | "-f" | "-L" | "-p" | "-S", path: string): boolean;
```

Comment:
Makes a lot of sense to me, but I would like to see the literals assigned to a separate `type` declaration. Test can possibly accept quite a number of options and having all of them in the function signature does not scale well.